base/MAPL_Resource.F90

```diff
@@ -0,0 +1,411 @@
+#include "MAPL_Exceptions.h"
+#include "MAPL_ErrLog.h"
+#include "unused_dummy.H"
+
+!=============================================================================
+
+
+module MAPL_ResourceMod
```

Comment:
My big question is whether we can come up with a better name for this module and associated procedures.  "Resource" is a bit vague.   We should debate on Thu.

Reply:
Sounds good. We can discuss what this module should encompass beyond `GetResource`

Reply:
Per MAPL refactoring meeting: Leaving the name the same for now.